doc/bibliography/openmodelica.bib

```diff
@@ -76,6 +76,17 @@ @BOOK{openmodelica.org:fritzson:2004
 %% [*2*] PhD/Lic theses
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@PHDTHESIS{openmodelica.org:lennart:phd:2016,
+  author = {Lennart Ochel},
+  title = {Petri-Netz-basierte Simulation biologischer Prozesse mit OpenModelica},
+  type = "{Doctoral thesis}",
+  year = {2016},
```

Comment:
It was published in 2017.

Reply:
@lochel but in the pdf file, it says December 2016

Reply:
https://pub.uni-bielefeld.de/record/2913956 says 2017. Link to that